### DISCUSSION TITLE
Is the context menu for cell outputs supposed to have copy/paste actions?

### DISCUSSION THREAD

**Discussion by starball5** [microsoft/vscode-jupyter#13904] [Questions and Answers]
What exactly was the resolution of #7575? Did it just add keyboard shortcuts for copy/paste? Or was it also supposed to add context menu items for it? Because I don't see context menu items for copy/paste. I noticed because of [this question on Stack Overflow](https://stackoverflow.com/q/76670422/11107541).

**Comment by Andrzej-Andrzej** [comment]
I am experiencing the same issue. This is basic functionality that should be allowed. How can it be repaired ?
